lib/createInlinePluginCreator.js

```diff
@@ -181,6 +181,11 @@ function createInlinePluginCreator(packages, multiContext, synchronizer, flags)
 		};
 
 		const prepare = async (pluginOptions, context) => {
+			if (pkg.options.sequentialPrepare) {
```

Comment:
We're going to separate **semrel** and **msr** flags in future. So let's use `flags.sequentialPrepare` here.